wiki/History_of_osu!/2014/id.md

```diff
@@ -1,29 +1,44 @@
----
-outdated_translation: true
----
+# Sejarah osu! 2014
 
-# 2014
+## Januari
+
+Sistem [*performance point* (pp)](/wiki/Performance_points) dinonaktifkan untuk proses pemeliharaan pada tanggal 20 Januari 2014 akibat kesalahan perhitungan yang terjadi pada beberapa bulan sebelumnya \[1\]. Penonaktifan tersebut diikuti oleh pengumuman pada tanggal 26 Januari 2014 yang menjelaskan seputar perombakan sistem pp secara menyeluruh \[2,3\]. Sistem baru ini dirancang dan diimplementasikan oleh [Tom94](https://osu.ppy.sh/users/1857058) dan bertujuan agar skor pp ke depannya dihitung berdasarkan nilai kesulitan ("*difficulty value*") yang terpisah dari [*star rating*](/wiki/Beatmapping/Star_rating) untuk seluruh kombinasi [*beatmap*](/wiki/Beatmap) dan [*mod*](/wiki/Game_modifier) yang berlaku. Sebagai dampaknya, sistem pp yang lama kemudian dikenal sebagai "ppv1" dan sistem pp yang baru dinamakan "ppv2".
 
 ## Maret
 
-Saat elemen skinning bertambah banyak, elemen, `playfield.jpg` dihapus dan diperbarui dan sekarang tidak dapat diubah. `playfield.jpg` adalah elemen yang memungkinkan Anda untuk menggunakan gambar apa pun sebagai latar belakang default beatmap, jika tidak ada yang disediakan oleh pembuat map.
+Pada 1 Maret 2014, sistem *performance point* untuk mode permainan [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), dan [osu!mania](/wiki/Game_mode/osu!mania) yang telah direncanakan berbulan-bulan sebelumnya oleh Tom94 resmi digunakan sebagai sistem peringkat resmi untuk ketiga mode permainan tersebut. Perubahan ini disertai dengan peringatan bahwa pengembangan sistem pp yang ada pada saat itu masih berjalan dan masukan lebih lanjut masih sangat dibutuhkan \[4\].
+
+Dari area *skinning*, elemen `playfield.jpg` yang sebelumnya dapat dimodifikasi resmi dipensiunkan dan diganti dengan gambar bawaan yang tidak dapat diubah. Elemen `playfield.jpg` merupakan elemen *skinning* yang memungkinkan para pemain untuk menggunakan gambar apa pun yang mereka inginkan sebagai latar belakang beatmap yang dimainkan apabila beatmap yang bersangkutan tidak memiliki latar belakang. Fungsionalitas ini kemudian diperkenalkan kembali melalui elemen baru `menu-background.jpg` yang mengubah tampilan gambar latar beatmap dan menu utama secara bersamaan \[5,6\].
+
+![](img/2014-03_01.jpg "Latar belakang bawaan untuk beatmap tanpa latar belakang yang spesifik*")
```

Comment:
is the stray `*` in tooltip intentional?

Reply:
oh the `*` should definitely not be there yeah